src/services/worker/cron/fetch.cron.ts

```diff
@@ -1,13 +1,13 @@
 import { Injectable } from '@nestjs/common';
 
-import { Cron } from '@nestjs/schedule';
+import {Cron, CronExpression} from '@nestjs/schedule';
 import { FetchTask } from '../tasks/fetch.task';
 
 @Injectable()
 export class FetchCron {
   constructor(private _fetchTask: FetchTask) {}
 
-  @Cron('* * * * * *')
+  @Cron(CronExpression.EVERY_SECOND)
```

Comment:
Since you're doing changes here, which is good.
Because 1 second is too small period of time, please add concurrency of 1 to not overlap executions in a slow network conditions

Reply:
Lets maybe do it in a seperate branch/PR?